federation/pkg/dnsprovider/providers/google/clouddns/rrsets.go

```diff
@@ -58,7 +58,7 @@ func (rrsets ResourceRecordSets) Add(rrset dnsprovider.ResourceRecordSet) (dnspr
 
 func (rrsets ResourceRecordSets) Remove(rrset dnsprovider.ResourceRecordSet) error {
 	service := rrsets.zone.zones.interface_.service.Changes()
-	deletions := []interfaces.ResourceRecordSet{rrset.(ResourceRecordSet).impl}
+	deletions := []interfaces.ResourceRecordSet{rrset.(*ResourceRecordSet).impl}
```

Comment:
Good catch, thanks.  I'm surprised that this was not caught by compile-time checks and unit tests.  I'll take a TODO to fix that.